### ISSUE TITLE
Filestore problems (of the web client) should be automatically fixed by Odoo

### ISSUE THREAD

**Issue by claudiostevanatocylix** [odoo/odoo#124153]
Impacted versions: 14.0

Steps to reproduce: Remove or destroy somehow the filestore (web client)

Current behavior: The page does not load css and js

Expected behavior: It should restore it automatically

Sometimes my filestore just stop working, installing an app or restoring a db without the filestore, I do not understand why Odoo throws warnings instead of recreating the js and css assets automatically, sometimes I lose hours trying to fix it.
I tried to remove assets from the db and restart everything -> still a page without css/js.

It is a shame that a dev has to lose time fixing these structural errors instead of develop.

In my honest opinion, odoo should automatically understand that the assets are somehow missing and rebuild them automatically, I understand why real attachments should throw errors/warnings, but why the hell css/js does not just works? It is absurd that a project so big still has these annoying practices.

**Comment by gdgellatly** [comment]
It actually does this. And it has a developer function that will also force this. And it is easy to just delete in script and it will also automatically rebuild these. And it has a dev mode which will ignore filestore/db for static files, views, reports etc. Most dev's are not losing time loading assets from filestore because they use the built in dev tools.

**Comment by claudiostevanatocylix** [comment]
> It actually does this. And it has a developer function that will also force this. And it is easy to just delete in script and it will also automatically rebuild these. And it has a dev mode which will ignore filestore/db for static files, views, reports etc. Most dev's are not losing time loading assets from filestore because they use the built in dev tools.

So how can I enable this dev mode? Do you refer to the developer mode in the GUI?

**Comment by vava-odoo** [comment]
Hi @claudiostevanatocylix 

Please find the documentation about developer/debug mode [here](https://www.odoo.com/documentation/16.0/applications/general/developer_mode.html).

Thanks @gdgellatly for the answer :+1: 

Did he fully answer your question @claudiostevanatocylix? Can I close the issue?


> @claudiostevanatocylix There is no point in reporting issues here. Odoo has 2000k reoorted never fixed issues.

@TadeuszKarpinski 2000k? Waw, that's a lot! ;-)

**Comment by gdgellatly** [comment]
@claudiostevanatocylix no, I was referring to --dev cli options at startup. Developer mode does allow regenerate assets but I never use it, I just have a cli command that deletes all the attachment records.
